### ISSUE TITLE
format(yml file, %{interpolator})

### ISSUE THREAD

**Issue by JDFortune** [rubocop/rubocop#10604]
I was just told to post this error message here:

`0.86.0 (using Parser 3.1.2.0, rubocop-ast 0.8.0, running on ruby 3.0.0 x86_64-linux)`

My code at one point references a yaml file:
using
```ruby
require 'yaml'
MESSAGES = YAML.load_file('rps_message.yml')
```
yml file:
`computer_choice: "Computer has chosen %{choice}"`

ruby file:
`puts format(MESSAGES['computer_choice'], choice: variable)`

the code works, but rubocop is pulling up an error (not a cop, but an error in the rubocop program itself)

-------------

## Expected behavior

Describe here how you expected RuboCop to behave in this particular situation.

## Actual behavior

Describe here what actually happened.
Please use `rubocop --debug` when pasting rubocop output as it contains additional information.

## Steps to reproduce the problem

This is extremely important! Providing us with a reliable way to reproduce
a problem will expedite its solution.

## RuboCop version

Include the output of `rubocop -V` or `bundle exec rubocop -V` if using Bundler.
If you see extension cop versions (e.g. `rubocop-performance`, `rubocop-rspec`, and others)
output by `rubocop -V`, include them as well. Here's an example:

```
$ [bundle exec] rubocop -V
1.29.0 (using Parser 2.7.2.0, rubocop-ast 1.1.1, running on ruby 2.7.2 x86_64-linux)
  - rubocop-performance 1.9.1
  - rubocop-rspec 2.0.0
```

**Comment by ydah** [comment]
Thank you so much.

Reproduced it.
If the error occurs in `Style/RaiseArgs` as shown below, please upgrade the RuboCop version to v0.89 to avoid the error.
Refs: https://github.com/rubocop/rubocop/releases/tag/v0.89.0

```
1 error occurred:
An error occurred while Style/RaiseArgs cop was inspecting foo/sample.rb:1:1.
Errors are usually caused by RuboCop bugs.
Please, report your problems to RuboCop's issue tracker.
https://github.com/rubocop/rubocop/issues

Mention the following information in the issue report:
1.28.2 (using Parser 3.1.1.0, rubocop-ast 1.17.0, running on ruby 2.7.4 x86_64-linux-gnu)
```

If it still does not improve, could you please include the following logs?

```
# example
An error occurred while Style/RaiseArgs cop was inspecting foo/sample.rb:1:1.
```

**Comment by ydah** [comment]
The reproduced code is as follows:

```ruby
# frozen_string_literal: true

require 'yaml'
MESSAGES = YAML.load_file('rps_message.yml')

puts format(MESSAGES['computer_choice'], choice: variable)
```

**Comment by fatkodima** [comment]
Can confirm that there was an error for the example in the previous comment and rubocop `0.86.0`, and now it is gone.
@koic You can close this issue.
